server/ai_mediaserver.go

```diff
@@ -859,6 +860,9 @@ func (ls *LivepeerServer) UpdateLiveVideo() http.Handler {
 			http.Error(w, err.Error(), http.StatusInternalServerError)
 			return
 		}
+
+		w.Header().Set("Access-Control-Allow-Origin", "*")
```

Comment:
Why not use the `corsHeaders` helper here? I am not sure we want to set this unconditionally.

Reply:
Good point. Any issue to add `GET` to `w.Header().Set("Access-Control-Allow-Methods", "OPTIONS, POST")` to cover /status endpoint?

Reply:
`UpdateLiveVideo` here is a POST, right?

I guess you probably need the GET for the status endpoint? Adding that globally is little sloppy but probably okay.  Although I see the code in the status endpoint specifically skips `corsHeaders` for an OPTIONS request, so  I'm not sure what adding the GET actually does.

Reply:
Ah well, i also had the if stmt checking for OPTIONS method backwards in status request. Thank you for pointing that out.

My main goal here is enabling spinning up a full go-livepeer stack locally with a web ui front end to create or test pipelines.  I could add a reverse proxy to do the cors I guess but we already had the `LIVE_AI_ALLOW_CORS` for the WHIP ingest so I was trying to enable updating and getting status of the stream as well.

Reply:
That's very cool, looking forward to seeing it! Not having cors for these endpoints in the first place was probably an oversight.